R/calcFE.R

```diff
@@ -59,6 +59,24 @@ calcFE <- function(source = "IEA", scenario_proj = "SSP2") {
     x <- mbind(x, setNames(x[, , "SE|Heat|Biomass|CHP (EJ/yr)"] +
       x[, , "SE|Heat|Biomass|HP (EJ/yr)"], "SE|Heat|Biomass (EJ/yr)"))
 
+
+    # add secondary energy totals
+    x <- mbind(x, setNames(x[, , "SE|Gases|Biomass (EJ/yr)"]
+                           + x[, , "SE|Gases|Fossil|Coal (EJ/yr)"]
+                           + x[, , "SE|Gases|Fossil|Natural Gas (EJ/yr)"], "SE|Gases (EJ/yr)"))
+
+    x <- mbind(x, setNames(x[, , "SE|Liquids|Bio Oil (EJ/yr)"]
```

Comment:
why not use the variable `SE|Liquids|Biomass`? Is `SE|Liquids|Bio Oil` equal to `SE|Liquids|Biomass`?

Reply:
`getNames(x[, , "SE|Liquids", pmatch=T])`  yields
`"SE|Liquids|Bio Oil (EJ/yr)"    "SE|Liquids|Fossil|Gas (EJ/yr)" "SE|Liquids|Fossil|Oil (EJ/yr)"` 

`SE|Liquids|Biomass`  seems to be not available in this context  (uses `structuremappingIO_reporting.csv`)

Reply:
This is actually what got me started again with the structuremappings....
https://github.com/remindmodel/development_issues/issues/83#issuecomment-1801368318

I don't think this is an output that is produced from structuremappingIO_reporting, and before changing that I want to make sure it builds up from the right structuremapping...